miss_hit_core/errors.py

```diff
@@ -197,6 +204,36 @@ def to_json(self):
                 "fixable"   : self.fixed,
                 "fatal"     : self.fatal}
 
+    def to_gitlab(self):
+        check_name = self.check_id if self.check_id else ""
+
+        map_severity = {
+            "low": "info",
+            "medium": "minor",
+            "high": "major",
+        }
+        severity = map_severity.get(self.severity, "minor")
+
+        if self.fatal:
+            severity = "critical"
+            check_name = "fatal"
+
+        contents = {
+            "description" : self.message,
+            "check_name"  : check_name,
```

Comment:
```suggestion
            "type": "issue",
            "check_name"  : check_name,
            "categories": ["Style"],
```
Eventually map categories in https://github.com/codeclimate/platform/blob/master/spec/analyzers/SPEC.md#data-types to `Message.kind`?